packages/logs/lib/models/messages.ts

```diff
@@ -128,20 +132,24 @@ export async function listOperations(opts: {
         });
     }
 
+    const cursor = opts.cursor ? parseCursor(opts.cursor) : undefined;
     const res = await client.search<OperationRow>({
         index: indexMessages.index,
         size: opts.limit,
-        sort: [{ createdAt: 'desc' }, '_score'],
+        sort: [{ createdAt: 'desc' }, 'id'],
         track_total_hits: true,
+        search_after: cursor,
         query
     });
     const hits = res.hits;
 
+    const total = typeof hits.total === 'object' ? hits.total.value : hits.hits.length;
```

Comment:
can we use the null coalescing operator here? `hits.total?.value ?? hits.hits.length;`